packages/autorest.typescript/test/rlcIntegration/generated/bodyComplexRest/src/outputModels.ts

```diff
@@ -164,7 +164,7 @@ export interface SharkOutputParent extends FishOutputParent {
 
 export interface SawsharkOutput extends SharkOutputParent {
   /** Value may contain base64 encoded characters */
-  picture?: string;
+  picture?: Uint8Array;
```

Comment:
@joheredi Even we change to `Uint8Array` but the real result is still a string returned from the core client